Dockerfile

```diff
@@ -16,9 +16,9 @@ RUN curl -s https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
     rm -rf ~/.cache/pip && \
     rm -f get-pip.py
 
-RUN Rscript -e "devtools::install_github('civisanalytics/civis-r', ref = 'v1.2.0', upgrade_dependencies = FALSE);"
+RUN Rscript -e "devtools::install_github('civisanalytics/civis-r', ref = 'v1.3.0', upgrade_dependencies = FALSE);"
 
-ENV VERSION=2.4.1 \
+ENV VERSION=2.4.2 \
```

Comment:
This update includes a minor version increase, so I think this should be a minor version increase in datascience-r as well.